colossalai/inference/batch_bucket.py

```diff
@@ -88,6 +91,23 @@ def is_compact(self):
             == torch.nonzero(self._block_tables[:, 0] >= 0).numel()
         )
 
+    @property
+    def use_spec_dec(self) -> bool:
+        return self._use_spec_dec
+
+    @property
+    def ver_num(self) -> int:
+        assert self.use_spec_dec and self._ver_num is not None
+        return self._ver_num
+
+    def set_use_spec_dec(self, n: int = 5) -> None:
+        self._use_spec_dec = True
+        self._ver_num = n
```

Comment:
give the argument n the same name as self._ver_num

Reply:
Renamed